### ISSUE TITLE
Ekaterina: Week 1, Asynchronous programming

### ISSUE THREAD

**Issue by ekaterinaasf** [HackYourFutureBelgium/class-7#196]
### Prep Work

- [ ] [javascript.info/settimeout-setinterval] (https://javascript.info/settimeout-setinterval)
- [ ] [event loop](http://latentflip.com/loupe/)
- [ ] [Beau from FCC](https://www.youtube.com/watch?v=kOcFZV3c75I)

### Name of Project:

- [repository](github.com/user-name/repo-name)
- [live demo](https://user-name.github.io/repo-name)
- [ ]  [step 1](link to branch for version 1)
- [ ]  [step 2](link to branch for version 2)
- [ ]  [step 3](link to branch for version 3)
- [ ]  ...

### Exercises:

- [ ] exercise name: [repo](https://github.com/user-name/repo-name), [live (if there is)]()
- [ ] exercise name: [repo](https://github.com/user-name/repo-name), [live (if there is)]()
- [ ] ...

> [Follow this link](https://github.com/HackYourFutureBelgium/class-6/issues/59) to study a well-done homework issue.

**Comment by ekaterinaasf** [comment]
Corrected few tests in examples in Closure. Not sure of necessary

**Comment by colevandersWands** [comment]
which tests did you correct?

**Comment by ekaterinaasf** [comment]
@colevandersWands 

Sorry for delay. Inside these files:
https://github.com/ekaterinaasf/closure/blob/master/exercises/4-sharing-state/example-1-pure-functions.js
https://github.com/ekaterinaasf/closure/blob/master/exercises/4-sharing-state/example-2-pure-closures.js

**Comment by colevandersWands** [comment]
good catch.  that was actually a big oversight on my part. those were supposed to be exercises, the nulls were supposed to be \_ 🤦.  

I reformatted these from last class and forgot to replace all of the nulls (they were they old \_). 
 updated: https://github.com/HackYourFutureBelgium/closure/tree/master/exercises/4-sharing-state

**Comment by ekaterinaasf** [comment]
https://github.com/ekaterinaasf/promises/tree/master/exercises/fill-in-the-blanks
There are few exercises (#6, 7, 8) where not all tests passed.

**Comment by colevandersWands** [comment]
6 & 7 - look into how to throw errors.  exercise 8 has an example of a thrown error that works

8 - give a closer look at your conditional check
